### PR TITLE
Resolve "Same style for both "Remove Branding" and "Google Analytics" in blog settings"

### LINKED ISSUE #1136: Same style for both "Remove Branding" and "Google Analytics" in blog settings

### PR DESCRIPTION
**Pull request by AmirSavand**
Closes #1136 

Also removed auto update for the "Remove Branding" setting.

Before:
image

After:
image

### CODE REVIEW

File: src/app/site/site_view.html [hunk 1]
```diff
@@ -425,23 +432,16 @@ <h5 class="panel-heading lower">
           <!-- Branding -->
           <div class="checkbox checkbox-primary checkbox-small">
             <input id="remove_branding" type="checkbox" ng-model="site.remove_branding"
-                   disable-on-request="removeBranding(site.remove_branding)"
                    title="{{ 'REMOVE_BRANDING_TITLE' | translate }}">
             <label for="remove_branding" translate>REMOVE_BRANDING</label>
+            <button class="btn btn-primary pull-right" type="button"
+                    disable-on-request="setCustomFooter(site.footer_text)">{{ 'SAVE' | translate }}
+            </button>
```
Comment (AmirSavand) [inline src/app/site/site_view.html]: Moved button up.

File: src/app/site/site_view.html [hunk 2]
```diff
@@ -425,23 +432,16 @@ <h5 class="panel-heading lower">
...
-            <div class="row">
-              <div class="col-md-10">
-                <textarea type="text" class="form-control" ng-model="site.footer_text"></textarea>
-              </div>
-              <div class="col-md-2 text-right">
-                <button class="btn btn-primary" type="button"
-                        disable-on-request="setCustomFooter(site.footer_text)">{{ 'SAVE' | translate }}
-                </button>
-              </div>
-            </div>
```
Comment (AmirSavand) [inline src/app/site/site_view.html]: Removed button from bottom.

**Comment by Alir3z4** [comment]
LGTM, @AmirSavand please merge this if nothing is left.

**Comment by AmirSavand** [comment]
@ArsalanSavand took long time to review, I merged.

**Pull request merged**